core/prisma/schema.prisma

```diff
@@ -192,7 +195,6 @@ model PubValue {
 
   // Add userId and integrationId to track who created the value
 
-  @@unique([pubId, relatedPubId, fieldId])
```

Comment:
Correct me if I'm wrong, but I think this is necessary because @allisonking did not remove this after altering the migrations manually?

Reply:
yes, I think that's right! although it's a little odd that the unique constraint can't be captured in the schema, it's just hanging out in the migrations... I think that should be okay though!